deploy/ocm-agent-operator-managednotifications/10-managednotifications-pdb.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: ocmagent.managed.openshift.io/v1alpha1
+kind: ManagedNotification
+metadata:
+  name: sre-pdb-managed-notifications
+  namespace: openshift-ocm-agent-operator
+spec:
+  notifications:
+    - activeBody: |-
+        The pod disruption budget is below the minimum disruptions allowed level and is not satisfied. The number of current healthy pods is less than the desired healthy pods. For more information, please refer to https://github.com/openshift/runbooks/blob/master/alerts/cluster-kube-controller-manager-operator/PodDisruptionBudgetLimit.md
```

Comment:
Are we giving the customer enough information to diagnose this? Are there fields included like the namespace and PDB we are referring to?

Reply:
i think what you are asking is answered in https://github.com/openshift/managed-cluster-config/pull/1827#issuecomment-1713556574

Reply:
So the answer is "no" right now? I think that's a prerequisite.